.github/workflows/reusable-generate-puml-svg.yaml

```diff
@@ -0,0 +1,50 @@
+#########################################################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#########################################################################################
+
+name: generate static PlantUML files
+on:
+  workflow_call:
+    # workflow outputs to generate-files job outputs
+    outputs:
+      puml-files:
+        description: "generated puml files"
+        value: ${{ jobs.generate-files.outputs.puml_files }}
+
+jobs:
+  generate-files:
+    runs-on: ubuntu-latest
+    # Map the job outputs to step outputs
+    outputs:
+      puml-files: ${{ steps.generate.generate.puml_files }}
+    steps:
+      - name: checkout source repo
+        uses: actions/checkout@v3
+
+      - name: Generate PlantUML Diagrams
+        id: generate
+        uses: rotaract/plantuml-action@v1.3.0
+        with:
+          format: svg
+          pattern: "**/*.puml"
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: my-puml-artifacts
```

Comment:
This name sounds a bit tutorial'isch. Why not just "artifacts". Also the name needs to be known by the ones using this workflow. So this should either be available as output, really easy to guess, or configurable as input. 

Same for the other workflow file

Reply:
[chore: rename artifacts](https://github.com/eclipse-tractusx/sig-infra/pull/186/commits/739ad3da9ea7ebd7e102cd2281e6fe4e58b3b4ae)

to verify that this also works i would start with `artifacts` name and create a little test workflow in sig-infra repo that this will also works as expected in etx, when this done i will update the reusable workflow with inputs.

Is this ok for the start? I guess need a bit more to present it in the office hour, so don´t want to rush but verify that this will also work in this organisation 😃